tcf_website/models/models.py

```diff
@@ -61,33 +73,62 @@ class Meta:
 
 
 class Subdepartment(models.Model):
-    """Subdepartment model.
+    """Subdepartment model. Represents a specific subject within a Department.
+    For example, "CREO" in the French department, "ENCW" in the English department,
+    or "CS - Computer Science" in the Computer Science department.
 
-    Belongs to a Department.
-    Has many Courses.
+    Relationships:
+    - Belongs to a Department.
+    - Has many Courses.
     """
-    # Subdepartment name. Required.
     name = models.CharField(max_length=255)
-    # Subdepartment description. Optional.
+    """The name of this Subdepartment. Required.
+
+    Examples:
+    - "Computer Science"
+    - "Religion - Judaism"
+
+    Considerations:
+    Some Subdepartments have descriptive names, like "Computer Science."
+    Other Subdepartments are named after their mnemonic, like "CREO."
```

Comment:
Might be important to mention that by far, _most_ subdepartments are named with their full name. The list of most acronyms is here:
![image](https://user-images.githubusercontent.com/55100084/108605491-5b526000-7382-11eb-8dfd-2de121aa323e.png)

Also LPPS, not sure why it's sorted in a different place. We also have several Unknown subdepartments; not sure what that's about off the top of my head.